easybuild/tools/toolchain/linalg.py

```diff
@@ -90,8 +90,9 @@ def set_variables(self):
         ## TODO is link order fully preserved with this order ?
         self._set_blas_variables()
         self._set_lapack_variables()
-        self._set_blacs_variables()
-        self._set_scalapack_variables()
+        if self.options.get('usempi', False):
```

Comment:
hmm, I feel we need a more direct check on whether or not the toolchain definition includes MPI or not, rather than relying on toolchain options...

Maybe check if the `MPI_MODULE_NAME` attribute is defined (and if it's not `None`)?

```python
if hasattr(self, 'MPI_MODULE_NAME') and getattr(self, 'MPI_MODULE_NAME') is not None:
    self._set_blacs_variables()
    self._set_scalapack_variables()
```